docs/install.md

```diff
@@ -695,8 +695,8 @@ Select the following tabs for the installation description of your target.
 
         <font size="+2">1. Download and extract the DietPi direct write image</font>
 
-        Download the **BIOS direct write image** `DietPi_NativePC-BIOS-x86_64-Bullseye.7z` from [`dietpi.com`](https://dietpi.com/#download) and
-        unzip the downloaded file to a local folder. It is a _7z_ archive format so you will need to install either [7zip for Windows](https://www.7-zip.org/) or other alternative tools.
+        Download the **Native PC for BIOS/CSM** > **Direct flash Image** from [`dietpi.com`](https://dietpi.com/#download) and
```

Comment:
On the download page it is called `Direct flash Image`, in the docs it is called `Direct write image`. Shall we align this? Actually `write` fits better as it is more generic, while `flashing` fits more for writing to an internal ROM or so, though common for SD cards for some reason.